2-mandatory/4-stocks.js

```diff
@@ -33,8 +33,38 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Solve the smaller problems, and then build those solutions back up to solve the larger problem.
         Functions can help with this!
 */
+function roundUpCalculation(rawNum) {
+    return Math.round((rawNum + Number.EPSILON) * 100) / 100;
+}
+
+function calculateAverage(sum, lengthOfList) {
+    return roundUpCalculation(sum / lengthOfList);
+}
+
+function sumOfPricesOverFiveDays(prices) {
+    let sum = 0;
+
+    for(let i = 0; i < prices.length; i++) {
+        sum += prices[i];
+    }
+
+    return sum;
+}
+
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    let averagePrices = [];
+
+    for(let i = 0; i < closingPricesForAllStocks.length; i++) {
+        let prices = closingPricesForAllStocks[i];
+
+        let sum = sumOfPricesOverFiveDays(prices);
+
+        let avg = calculateAverage(sum, prices.length);
```

Comment:
excellently named functions and such a good job breaking this down into smaller problems!